### DISCUSSION TITLE
Getting stock price at regular interval during trading hours

### DISCUSSION THREAD

**Discussion by charleslcso** [ghostfolio/ghostfolio#1154] [Q&A]
Hello, just set up an instance and am looking around. Wondering if it is possible to get stock prices from somewhere regularly during local trading hours, and update my portfolio so that I can see if I'm at loss or at gain. 

Just wondering.

**Comment by dtslvr** [comment]
Hi @charleslcso

I'm glad your instance is running. I think this is exactly the point of the project: To see the performance / net worth in (almost) real time. Maybe you want to check the example portfolio with all the current possibilities of Ghostfolio: https://ghostfol.io/en/demo
